packages/storybook/stories/va-alert.stories.jsx

```diff
@@ -30,8 +31,11 @@ export default {
   parameters: {
     docs: {
       description: {
-        component: `Use a heading element with an attribute named slot and a value of "headline" to control what is displayed for the alert's headline. 
-        Any children passed into this component without a parent slot "headline" will render in the alert's body.`,
+        component:
+          `Use a heading element with an attribute named slot and a value of "headline" to control what is displayed for the alert's headline. 
+        Any children passed into this component without a parent slot "headline" will render in the alert's body. 
+        This component has 2 events: close and vaComponentDidLoad.` +
```

Comment:
And should we also be calling out the analytics event not necessarily as something people should have to handle, but just to inform them about it?

![image](https://user-images.githubusercontent.com/2008881/151039373-b0b966d2-34fe-4e25-8d1b-99928cb55b72.png)


![image](https://user-images.githubusercontent.com/2008881/151039301-1e79e65c-a9f1-485b-88a8-e5f8c038f551.png)

Reply:
I removed the filter for component-library-analytics, so all events are displayed now. I left it out initially, but I think it's okay to display for the sake of completeness.